include/core/utils/hashset/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Do we really need to include this file twice and have it in two different locations? Note that it doesn't even list a copyright holder...

Reply:
Actually, I don't think we need to include either LICENSE file. The source files themselves point to the license terms online.